src/WallToolPaths.cpp

```diff
@@ -82,24 +88,116 @@ const VariableWidthPaths& WallToolPaths::getToolPaths()
     return toolpaths;
 }
 
+void WallToolPaths::computeInnerContour()
+{
+    //We'll remove all 0-width paths from the original toolpaths and store them separately as polygons.
+    VariableWidthPaths actual_toolpaths;
+    actual_toolpaths.reserve(toolpaths.size()); //A bit too much, but the correct order of magnitude.
+    VariableWidthPaths contour_paths;
+    contour_paths.reserve(toolpaths.size() / inset_count);
```

Comment:
Implicit conversion of `inset_count` use `static_cast<size_t>(inset_count)`

Reply:
No, `inset_count` should not be a coordinate type. It's a count, so it should be `size_t`. I've changed that here: 3c03305fe73ff4a97ee78e46273aee8620628091

I had already seen this when I changed the member variables to be non-const, but didn't take the effort to refactor that then. But here you go.

Reply:
[nerd alert:] While I agree in this case (and most cases in fact) `size_t` is the 'pointer-size' type ... it is often _used_ as the 'counter' type as well, because lists and vectors are max that size, but it doesn't _always_ make sense to equate the two (especially if the count rarely or never refers to a C++ sequence-object) ... I recently had an example where it made more sense to use `coord_t` anyway in my mind at least.